builtInFunctions/esdtDataStorage.go

```diff
@@ -292,12 +356,20 @@ func (e *esdtDataStorage) saveESDTMetaDataToSystemAccount(
 		TokenMetaData: esdtData.TokenMetaData,
 		Properties:    make([]byte, e.shardCoordinator.NumberOfShards()),
 	}
-	selfID := e.shardCoordinator.SelfId()
-	if selfID != core.MetachainShardId {
-		esdtDataOnSystemAcc.Properties[selfID] = existsOnShard
+	if len(currentSaveData) == 0 && e.flagSendAlwaysEnableEpoch.IsSet() {
+		esdtDataOnSystemAcc.Properties = nil
+		esdtDataOnSystemAcc.Reserved = []byte{1}
+		esdtDataOnSystemAcc.Value.Set(esdtData.Value)
 	}
-	if senderShardID != core.MetachainShardId {
-		esdtDataOnSystemAcc.Properties[senderShardID] = existsOnShard
+
+	if !e.flagSendAlwaysEnableEpoch.IsSet() {
```

Comment:
`always send meta data` sounds better than `send metadata always`. any way we can change this?